themes/custom/droopler_theme/droopler_theme.theme

```diff
@@ -420,6 +428,9 @@ function droopler_theme_is_special_route() {
 function droopler_theme_show_title_block() {
   $route = \Drupal::routeMatch();
   $route_name = $route->getRouteName();
+  if ($route_name == 'entity.commerce_product.canonical') {
+    return !droopler_theme_check_is_droopler_products();
```

Comment:
Jak wyżej, nie zostawiałbym w kodzie PHP skórki odniesień do d_commerce

Reply:
Proponuje to zostawić na ten moment. Próbowałem to przenieść do modułu i działało ok dla commerce_product, content_page i block ale np. zniknał tytuł na stronie logowania. Problem z tymi tytułami już kiedyś był i pasowałoby to zrobić od nowa w osobnym zadaniu.

Reply:
hmmm, wydaje mi się że nic się nie zmieni jeśli zrobimy hooka/altera który przyjmie $route i zwróci TRUE/FALSE w zależności od tego, czy tytuł ma być pokazany. Wtedy jeśli którykolwiek moduł zwróci FALSE to chowamy tytuł.